lib/ansible/modules/files/find.py

```diff
@@ -250,19 +250,22 @@ def sizefilter(st, size):
 
 
 def contentfilter(fsname, pattern):
-    '''filter files which contain the given expression'''
+    '''filter files which contain the given expression
+    :param fsname:
+    :param pattern:
+    :rtype: bool
+    '''
```

Comment:
If you're going to add docstrings, note that we use sphinx format for them:
``` python
"""
Filter files which contain the given expression
:arg  fsname: A filename to scan for lines matching a pattern
:arg pattern: The pattern to look for inside of the file.
:rtype: bool
:returns: True if one of the lines in fsname matched the pattern.  Otherwise, False.
"""